### ISSUE TITLE
FMS/Turtlebot connection tests

### ISSUE THREAD

**Issue by JonasPetersenFAU** [amosproj/amos2022ss03-turtlebot-fleet-management#61]
## User story
1. As a SD
2. I want to run the first connection attempt between the 2 devices
3. So that we can try ifthe connection works

## Acceptance criteria
* Connection between devices is established

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release

**Comment by Sebastian2023** [comment]
MQTT connection between fleet management software and turtlebot works in both directions. VDA5050 (de)serialization works on turtlebot side as well.

![MQTT](https://user-images.githubusercontent.com/92729241/171201827-2fff224d-6373-44e5-b84b-581f1de24544.png)
